components/Toggle/index.d.ts

```diff
@@ -1,13 +1,14 @@
 import React from 'react';
 
 export interface ToggleProps {
-    id?: string;
-    checked?: boolean;
-    theme?: {
-        colors?: object;
-        spacing?: object;
-        baseFontSize?: number;
-    };
+  onChange?: React.ChangeEventHandler<HTMLInputElement>;
```

Comment:
i think this prop `onChange` should be in the `Toggle` component too.
I noticed that component prop only stays here.
Or i missing something?

Reply:
Do you mean to add it to the Proptypes too?